test/verify/check-machines-dbus

```diff
@@ -296,6 +308,10 @@ class TestMachinesDBus(machineslib.TestMachines):
         # Check changed boot order have persisted
         b.wait_text_not("#vm-VmNotInstalled-boot-order", bootOrder)
 
+        # Check firmware changes persisted
+        if float(m.execute("virsh --version").strip()[:3]) >= 5.2:
```

Comment:
Please remember the "does this support EFI" check from above in a boolean, and re-use it here.